src/Pos/CompileConfig/Type.hs

```diff
@@ -42,6 +40,8 @@ data CompileConfig = CompileConfig
       -- ^ VSS certificates min timeout to live (number of epochs)
     , ccLightDlgConfirmationTimeout   :: !Int
       -- ^ Timeout for holding light psks confirmations
+    , ccDlgCacheParam                 :: !Int
```

Comment:
I understand the convention for sure, but still comment is useless as it is now.